docs/newContributors.md

```diff
@@ -0,0 +1,86 @@
+<h1 align="center">Hello Peeps 👋</h1>
+<h2 align="center">Welcome to the <a href="https://github.com/siv2r/kidney-exchange">kidney-exchange</a> Repo 🎉</h2>
+
+<h2 align="center">Here, is the Complete Guide to Contribute to this Repo.</h2>
+
+
+
+ 1.  Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+      > **Open Source is more about Collaboration.** 
+
+    
+
+ 2.  So to start Contributing to our Project you can solve existing issues or if you have found any bugs or want to add a new feature to Project you can create New Issue, that's purely your choice.
+
+
+
+  ![](../images/Issues.png)
+
+
+
+ 3.  Project Maintainers and Mentors will guide you with the issue and incase of any problem you face, you can take help from the community in the discord server.
+
+
+ 4.  So, Now you have an issue to work upon. So first you need is to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way:-
+
+
+
+  ![](../images/Fork.png)
+
+
+ 5.  Clone the Repo you have forked:-
+
+
+  
+      - Copy the link of the forked Repo.
+      
+      - Go to the desired location in the terminal, where you want to place the project.
+      
+      - Run command ``git clone link_of_forked_repo``.
+      
+      - Run Command ``cd kidney-exchange ``.
+      
+      - Open the folder with your favorite text editor command.
+
+
+
+ 6.  Set upstream to the Repo Using Command  `` git remote add upstream https://github.com/siv2r/kidney-exchange``.
+  
+
+
+ 7.  Make a branch using the command  ``git checkout -b branch_name ``.
+
+
+
+ 8.  Now it is the time to make required changes in the project 🌸
```

Comment:
Can you also add the following line?

You can look through the [good first issues](https://github.com/siv2r/kidney-exchange/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and select the one that you like to work on. If your interested issue is already assigned to someone, you can also collaborate to get it done. 

Also, add the following as point no 9.
If none of the issues matches your interest or things are a bit overwhelming, you can ask for guidance in the discord group.

Reply:
his should be 2or 3 point as I have included issues point ther

Reply:
What do u mean by issues points?

Reply:
@siv2r  I mean that  Points related to the issue are mentioned in Starting.

Reply:
Ahh, I get it now. Then can you add these to the start. 

>You can look through the good first issues and select the one that you like to work on. If your interested issue is already assigned to someone, you can also collaborate to get it done.

Reply:
Ook I will try do it by today, Sorry for being late.

Reply:
No worries, take your time. There is no time constrain here.

Reply:
Also, please click the button `Resolve conversation` after implementing the change. It helps to keep track of the progress.